config/runtime.exs

```diff
@@ -196,6 +196,13 @@ RuntimeConfig.config_env_try do
   end
 end
 
+case RuntimeConfig.get_env_as_non_neg_integer("PORTABILLING_ACCOUNT_PASSWORD_LENGTH") do
+  account_password_length ->
```

Comment:
Please use a "full" local variable name, as it is done in other functions (`portabilling_account_password_length`).